Defs/Ammo/Neolithic/SlingBullet.xml

```diff
@@ -1,213 +1,173 @@
+    </ThingCategoryDef>
+	
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_SlingBullet</defName>
+		<label>sling bullets</label>
+		<ammoTypes>
+			<Ammo_SlingBullet_Stone>Projectile_SlingBullet_Stone</Ammo_SlingBullet_Stone>
+			<Ammo_SlingBullet_Steel>Projectile_SlingBullet_Steel</Ammo_SlingBullet_Steel>
+		</ammoTypes>
+	</CombatExtended.AmmoSetDef>
+	
+	<!-- ==================== Ammo ========================== -->
+
+  <ThingDef Class="CombatExtended.AmmoDef" Name="AmmoSlingBulletBase" ParentName="NeolithicAmmoBase" Abstract="True">
+    <description>An aerodynamic stone or metal pellet. Perfect for being launched by a sling.</description>
+    <statBases>
+      <Mass>0.03</Mass>
+      <Bulk>0.07</Bulk>
+    </statBases>
+    <thingCategories>
+      <li>AmmoSlingBullet</li>
+    </thingCategories>
+    <stackLimit>5000</stackLimit>
+    <tradeTags>
+      <li>CE_AutoEnableTrade</li>
+      <li>CE_AutoEnableCrafting_FueledSmithy</li>
+      <li>CE_AutoEnableCrafting_ElectricSmithy</li>
+    </tradeTags>
+  </ThingDef>
+
+  <ThingDef Class="CombatExtended.AmmoDef" ParentName="AmmoSlingBulletBase">
+    <defName>Ammo_SlingBullet_Stone</defName>
+    <label>sling bullet (stone)</label>
+    <graphicData>
+      <texPath>Things/Ammo/Neolithic/SlingBullet/Stone</texPath>
+      <graphicClass>Graphic_StackCount</graphicClass>
+    </graphicData>
+    <statBases>
+      <MarketValue>0.1</MarketValue>
+    </statBases>
+		<ammoClass>StoneSlingBullet</ammoClass>
+    <tradeTags>
+      <li>CE_AutoEnableCrafting_CraftingSpot</li>
+    </tradeTags>
+  </ThingDef>
+
+  <ThingDef Class="CombatExtended.AmmoDef" ParentName="AmmoSlingBulletBase">
+    <defName>Ammo_SlingBullet_Steel</defName>
+    <label>sling bullet (steel)</label>
+    <graphicData>
+      <texPath>Things/Ammo/Neolithic/SlingBullet/Steel</texPath>
+      <graphicClass>Graphic_StackCount</graphicClass>
+    </graphicData>
+    <statBases>
+      <Mass>0.085</Mass>	
+      <MarketValue>0.19</MarketValue>
+    </statBases>
+		<ammoClass>SteelSlingBullet</ammoClass>
+  </ThingDef>
+	
+	<!-- ================== Projectiles ================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" Name="BaseSlingBullet" ParentName="BaseBullet" Abstract="true">
+		<graphicData>
+			<texPath>Things/Mote/Stone</texPath>
+			<graphicClass>Graphic_Single</graphicClass>
+		</graphicData>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageDef>Blunt</damageDef>
+			<speed>9</speed>
+		</projectile>
+	</ThingDef>
+
+	<!-- The amount of arrows usable over the ammo lifetime for each resource cost is equal to
+		[r / (1 - x)];
+			x = preExplosionSpawnChance,
+			r = resource cost. -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="BaseSlingBullet">
+		<defName>Projectile_SlingBullet_Stone</defName>
+		<label>sling bullet (stone)</label>
+		<graphicData>
+			<texPath>Projectiles/Sling/Stone</texPath>

```

Comment:
Missing texture at this path.

Reply:
Uploaded new projectile texture, update path.